### ISSUE TITLE
Markdown: strikethrough text does not work with the code span

### ISSUE THREAD

**Issue by tuxmartin** [RocketChat/fuselage#1213]
**Original issue: https://github.com/RocketChat/Rocket.Chat/issues/30185** :+1: 

I have my application that works as a notificator. It gets alerts from Zabbix and sends them to Rocket.Chat. After the alert is closed, my notification edits Rocket.Chat message.

For new alert message is:

```
:sos: WARNING `[server.example.net]: less than 10% 4.48 GB on volume /` [12345](https://zabbix.example.net/tr_events.php?triggerid=29634&eventid=247154493)
```

it is ok:
![image](https://github.com/RocketChat/fuselage/assets/1219067/c455d037-f9ab-4d44-99d6-2720675e6a7a)

but after the notificator app edit message, formatting breaks. Edited message:
```
:white_check_mark: ~~_WARNING `[server.example.net]: less than 10% 4.48 GB on volume /` Duration: 01:10:01 [12345](https://zabbix.example.net/tr_events.php?triggerid=29634&eventid=247154493)_~~

or

:white_check_mark: ~_WARNING `[server.example.net]: less than 10% 4.48 GB on volume /` Duration: 01:10:01 [12345](https://zabbix.example.net/tr_events.php?triggerid=29634&eventid=247154493)_~
```
and it is broken:
![image](https://github.com/RocketChat/fuselage/assets/1219067/f03d8b08-e21d-429f-99d9-eff29746de41)

if I put exactly the same markdown to the Visual Studio Code, it is ok:
![image](https://github.com/RocketChat/fuselage/assets/1219067/65230986-84cf-4f17-b3e5-0cc5aa4683c8)

Expected behavior: Show the message as in VS code editor

**@AdityaRaut13 found problem: https://github.com/RocketChat/Rocket.Chat/issues/30185#issuecomment-1750130282**

**Comment by tuxmartin** [comment]
[Info](https://github.com/RocketChat/Rocket.Chat/issues/30185#issuecomment-1788994471) from @AdityaRaut13:

> Ask about adding inLineCode to the italicContentBlock.
> You can also ask them to add it to other contentBlock .

**Comment by tuxmartin** [comment]
It's been broken for almost four months. There is a way to fix it. But still nothing happens :-(

**Comment by Subhojit-Dey1234** [comment]
@tuxmartin Raised the following PRs for this issue. Hope it will fix it. 
https://github.com/RocketChat/fuselage/pull/1256
https://github.com/RocketChat/Rocket.Chat/pull/31349

**Comment by dougfabris** [comment]
Hi there, thanks for the contribution! 🚀 💯 

Sorry, this error does not happen on the latest and it was fixed here: #1256

[Rocket.Chat](https://rocket.chat/)  provides support for, at least, 3 months for each release and 6 months for the latest releases of each major version

I would suggest you update your Rocket.Chat version to the latest stable version, which you can find in [Releases Page](https://github.com/RocketChat/Rocket.Chat/releases) and, check it [HERE](https://docs.rocket.chat/resources/get-support/enterprise-support#rocket.chat-versions) the table with versions and End of Life dates. 

---

**Questions? Help needed? Feature Requests?** 

- Join our [Open Server](https://open.rocket.chat/) in the #support channel and feel free to raise a question
- Join our [Community Forum](https://forums.rocket.chat/) and search/create a post there
